doc/PlayerBot/INSTALL.txt

```diff
@@ -0,0 +1,51 @@
+Installation:
+=============
+Merge PlayerbotAI code with your favorite core/mods, or simply compile. Follow CMangos Core installation steps for installation and building.
+
+Copy src/game/PlayerBot/playerbot.conf.dist.in for configuration variables!
+Install appropriate files from sql/PlayerBot
+
+Developers:
+===========
+When updating the playerbot.conf version, keep in mind the following locations (YYYYMMDDVV = Year, Month, Day, Version (version starting at 01, counting up as releases happen on the same day)):
+* src/game/PlayerBot/config.h
+define PLAYERBOT_CONF_VERSION    YYYYMMDDVV
+* src/game/PlayerBot/config.h.in
+define PLAYERBOT_CONF_VERSION    YYYYMMDDVV
+* src/game/PlayerBot/playerbot.conf.dist.in
+ConfVersion=YYYYMMDDVV
+* Of course don't forget to update your server's playerbot.conf.
+
+When updating the sql file(s), keep in mind the following:
+
+=== File Name Description in [sql/PlayerBot] folder ===
+
+    FULL_playerbotai_characters_r2_update
+    |    |           |          |  |
+    |    |           |          |  Only if the sql updates a previous sql
+    |    |           |          |
+    |    |           |          Playerbot revision related to sql
+    |    |           |
+    |    |           |Name of affected DB (default recommended name)
+    |    |
+    |    Playerbot sql file..drrr!
+    |
+    For clean install, just apply this sql
+
+* src/game/PlayerBot/revision_sql.h
```

Comment:
we have to remove playerbot revision and make it fully part of the core. (add it to mangos.sql)